frontend/components/queries/queryResults/AwaitingResults/_styles.scss

```diff
@@ -0,0 +1,22 @@
+.awaiting-results {
+  margin: 120px auto;
```

Comment:
Can we add more space to the left of this component? 


https://user-images.githubusercontent.com/71795832/194648610-39e7322f-5209-4a46-9212-829fcdef825f.mov